src/Bridges/Variable/flip_sign.jl

```diff
@@ -48,6 +48,9 @@ function MOI.delete(model::MOI.ModelLike, bridge::FlipSignBridge, i::IndexInVect
 end
 
 # Attributes, Bridge acting as a constraint
+function MOI.get(model::MOI.ModelLike, attr::MOI.ConstraintSet, bridge::FlipSignBridge)
+    return MOI.Nonpositives(length(bridge.flipped_variables))
```

Comment:
This is only valid for `NonposToNonnegBridge`, you can do
```julia
..., bridge::FlipSignBridge{T, S1}) where {T, S1}
return S1(...)
```

Reply:
Yeah cool, that would much nicer.

Reply:
But I think abstract type `FlipSignBridge{T, S1}` has only `NonposToNonnegBridge` as a subtype, there is no `NonnegToNonposBridge`.So, I think using `MOI.Nonpositives(length(bridge.flipped_variables))` would also be ok.